instrumentation/netty-reactor-0.8.0/README.md

```diff
@@ -0,0 +1,36 @@
+# Reactor Netty Instrumentation
+
+Instrumentation for Reactor Netty server and also some widely used Reactor Core library code.
+
+This module is largely responsible for instrumenting the Reactor Core library to facilitate the passing, retrieval,
+and linking of `Tokens` across contexts to tie asynchronous threads together for individual `Transactions`.
+
+This instrumentation is dependent on other instrumentation modules to start a `Transaction`.
+Typically, the `netty-n.n` modules work with this instrumentation and will start a `Transaction` (see `NettyDispatcher#channelRead`).
+
+Most commonly this instrumentation comes into play with SpringBoot usage, in which case the `spring` and `spring-webflux` 
+instrumentation modules also apply and should result in `Transactions` being renamed after the Spring controller.
```

Comment:
What do you think about calling out here in the README the Tokens mentioned earlier are created by ServerWebExchange in Spring WebFlux instrumentation? 

Or, there is already a comment in the code for Netty Reactor TokenLinkingSubscriber.  I think that comment should mention ServerWebExchange so the next person doesn't have to wonder about where that token comes from.

Reply:
Yea, I think the readme definitely has room for improvement, especially around the behavior of how the interactions between instrumentation modules work. I need to dive into that a bit deeper.

Reply:
Updated the comment in `TokenLinkingSubscriber` to specifically mention the token being created in `ServerWebExchange`.